packages/web/src/js/__tests__/FileUploader.test.ts

```diff
@@ -289,4 +289,44 @@ describe('FileUploader', () => {
       );
     });
   });
+
+  describe('findValidationTextInElement', () => {
+    it('should return true if validation text is found in any <li> element', () => {
+      const element = document.createElement('div');
```

Comment:
You can use the `beforeEach` test helper to create the element every time before the test run and not to create this in the test itself.